src/main/java/spireMapOverhaul/zones/manasurge/modifiers/common/negative/FeebleModifier.java

```diff
@@ -38,13 +41,13 @@ public void onUse(AbstractCard card, AbstractCreature target, UseCardAction acti
     @Override
     public List<TooltipInfo> additionalTooltips(AbstractCard card) {
         List<TooltipInfo> tooltips = new ArrayList<>();
-        tooltips.add(new TooltipInfo("[#ff8cd5]Feeble[] [#ff8cd5]|[] [#ff8cd5]Blight[]", "Gain #b1 #yFrail."));
+        tooltips.add(new TooltipInfo(ManaSurgeZone.getKeywordProper(ID) + " [#ff8cd5]|[] " + ManaSurgeZone.getKeywordProper(ManaSurgeZone.NEGATIVE_MOD), ManaSurgeZone.getKeywordDescription(ID)));
         return tooltips;
     }
 
     @Override
     public String modifyDescription(String rawDescription, AbstractCard card) {
-        return "[#ff8cd5]Feeble[]." + " NL " + rawDescription;
+        return ManaSurgeZone.getKeywordProper(ID) + ". NL " + rawDescription;
```

Comment:
```suggestion
        return ManaSurgeZone.getKeywordProper(ID) + LocalizedStrings.PERIOD + " NL " + rawDescription;
```
ZHS has other periods than we do, haha.

Reply:
zhs? fixed tho